torch/optim/lr_scheduler.py

```diff
@@ -1214,17 +1214,19 @@ def __init__(self,
         self.gamma = gamma
 
         if scale_fn is None:
+            self.scale_fn_custom = None      
```

Comment:
Can you name these `self._scale_fn_custom` since it should be private (same for `_scale_fn_ref`.